airflow/providers/airbyte/operators/airbyte.py

```diff
@@ -31,23 +31,36 @@ class AirbyteTriggerSyncOperator(BaseOperator):
     :type airbyte_conn_id: str
     :param connection_id: Required. The Airbyte ConnectionId UUID between a source and destination
     :type connection_id: str
+
     :param timeout: Optional. The amount of time, in seconds, to wait for the request to complete.
     :type timeout: float
     """
 
     @apply_defaults
     def __init__(
-        self, airbyte_conn_id: str, connection_id: str, timeout: Optional[int] = 3600, **kwargs
+        self,
+        airbyte_conn_id: str,
```

Comment:
```suggestion
        airbyte_conn_id: str = "default_airbyte_conn",
```

Most of operators and hooks usually have default conn_id